npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -120,51 +119,44 @@ export async function generateTsConfig (devServerConfig: AngularWebpackDevServer
 
   const specPattern = Array.isArray(cypressConfig.specPattern) ? cypressConfig.specPattern : [cypressConfig.specPattern]
 
-  const getProjectFilePath = (...fileParts: string[]): string => toPosix(path.join(...fileParts))
-
-  const includePaths = [...specPattern.map((pattern) => getProjectFilePath(projectRoot, pattern))]
+  const includePaths = [...specPattern]
 
   if (cypressConfig.supportFile) {
-    includePaths.push(toPosix(cypressConfig.supportFile))
+    const supportPath = toPosix(path.relative(workspaceRoot, cypressConfig.supportFile))
```

Comment:
This path being generated in an nx workspace creates a wrong path.

in an nx workspace we setup cypress in a sub directory of the project root.
i.e. 
```tree
./ (git root)
└──
   apps/
   └── angular-app/
      ├── cypress/
      │  └── tsconfig.json
      ├── src/
      ├── cypress.config.ts
      ├── project.json
      ├── tsconfig.app.json
      ├── tsconfig.cypress.json
      ├── tsconfig.editor.json
      ├── tsconfig.json
      └── tsconfig.spec.json
```
the current impl, creates this tsconfig
```json
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "outDir": "./out-tsc/cy",
    "allowSyntheticDefaultImports": true,
    "skipLibCheck": true,
    "types": ["cypress"]
  },
  "include": [
    "src/**/*.cy.ts",
    "src/**/*.cy.js",
    "apps/angular-app/cypress/support/component.ts"
  ]
}

```
which is invalid causing type errors when building form the missing `component.ts` file

The value would need to be 
`"cypress/support/component.ts"`

I believe in angular cli workspace the workspaceRoot and projectRoot are the same values, but in nx they are not. So maybe using projectRoot here instead.